cucumber/authenticators/features/support/authn_oidc_helper.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+#
+# Utility methods for OIDC authenticator
+#
+module AuthnOidcHelper
+  include AuthenticatorHelpers
+
+  # relevant for original oidc flow
+  # def login_with_oidc(service_id:, account:)
+  #   path = "#{conjur_hostname}/authn-oidc/#{service_id}/#{account}/login"
+  #   payload = { code: oidc_auth_code, redirect_uri: oidc_redirect_uri }
+  #   post(path, payload)
+  #   @login_oidc_conjur_token = @response_body
+  # end
+
+  # # relevant for oidc flow for Conjur oidc token retrieved in oidc login flow
+  # def authenticate_conjur_oidc_token_with_oidc(service_id:, account:)
+  #   path = "#{conjur_hostname}/authn-oidc/#{service_id}/#{account}/authenticate"
+  #   # TODO: Since the input going to change to a base64 signed token, i didnt invest time to extract the real values
+  #   payload = { id_token_encrypted: "login_oidc_conjur_token", user_name: "alice", expiration_time: "1231" }
+  #   post(path, payload)
+  # end
+
+  def authenticate_id_token_with_oidc(service_id:, account:)
+    path = "#{conjur_hostname}/authn-oidc/#{service_id}/#{account}/authenticate"
+    # TODO: Enable the following comment once the production code knows to get encoded id_token
+    # payload = { id_token: "#{oidc_id_token}" }
+    payload = { id_token: "{\"preferred_username\": \"alice\",\"email\": \"alice@conjur.net\"}" }
+    post(path, payload)
+  end
+
+  def oidc_authorization_code
+    path_script = "/authn-oidc/phantomjs/scripts/fetchAuthCode"
+    authorization_code_file = "cat /authn-oidc/phantomjs/scripts/authorization_code"
```

Comment:
I would remove this temp variable and inline `cat /authn-oidc/phantomjs/scripts/authorization_code` in the assignment on line 37 below.

This is especially true because the name is misleading: it's actually "the command to dump the file contents" and the "file" itself.  But the best name of all is the one you don't have to write :)

Reply:
Fix in pr #925